api/src/main/java/jakarta/security/enterprise/authentication/mechanism/http/HttpAuthenticationMechanismHandler.java

```diff
@@ -70,6 +70,8 @@
  * <p>
  * Applications do not need to supply an {@code HttpAuthenticationMechanismHandler} unless application-specific
  * behavior is desired.
+ *
+ *  @since 4.0
```

Comment:
Strange that oracle keeps (c) since 2015 in that case.

Reply:
It may look strange, but the class copies the javadoc descriptions from HttpAuthenticationMechanism. I thought long and hard to maybe let one inherit from the other, and at some point I had a design and implementation doing just that.

But in the end I decided it's clearer to not have them inherit from each other, so they are distinct types (just like IdentityStore and IdentityStoreHandler).

So since they code was copied, I retained the copyright from 2015.